cypress/components/number/number.cy.tsx

```diff
@@ -232,11 +233,11 @@ context("Tests for Number component", () => {
       }
     );
 
-    it.each([
+    it.each(([
```

Comment:
The reason we are getting an error without the `as unknown` type assertion, is because the `labelWidth` and `inputWidth` props actually expect numbers rather than strings, so this should work:

```tsx
it.each([
   [10, 90, 135, 1229],
   [30, 70, 409, 956],
   [80, 20, 1092, 273],
] as [NumberProps["labelWidth"], NumberProps["inputWidth"], number, number][])(
```